### PR TITLE
Fully automate Hellknight Signifier Dedication

### LINKED ISSUE #4447: Hellknight Signifier Dedication bonus mod
The below bonus is only implemented for Deception not for Intimidation.
While wearing your signifer’s mask, you gain +1 circumstance bonus to Deception checks to Lie, Intimidation checks, and Deception DCs against Sense Motive.

### PR DESCRIPTION
**Pull request by TikaelSol**
Closes #4447 
Supersedes #4456

### CODE REVIEW

File: packs/data/feats.db/hellknight-signifer-dedication.json [hunk 1]
```diff
@@ -44,19 +44,61 @@
...
+                                "not": "skill:arc:rank:2"
+                            }
+                        ],
+                        "value": "arc"
+                    },
+                    {
+                        "label": "PF2E.SkillNat",
+                        "predicate": [
+                            {
+                                "not": "skill:nat:rank:2"
```
Comment (stwlam) [inline packs/data/feats.db/hellknight-signifer-dedication.json:69]: but what if master

File: packs/data/feats.db/hellknight-signifer-dedication.json [hunk 2]
```diff
@@ -44,19 +44,61 @@
...
+                    },
+                    {
+                        "label": "PF2E.SkillOcc",
+                        "predicate": [
+                            {
+                                "not": "skill:occ:rank:2"
+                            }
+                        ],
+                        "value": "occ"
+                    }
```
Comment (TikaelSol) [inline packs/data/feats.db/hellknight-signifer-dedication.json:91]:
```suggestion
                            {
                                "lt": ["skill:arc:rank",2]
                            }
                        ],
                        "value": "arc"
                    },
                    {
                        "label": "PF2E.SkillNat",
                        "predicate": [
                            {
                                "lt": ["skill:nat:rank",2]
                            }
                        ],
                        "value": "nat"
                    },
                    {
                        "label": "PF2E.SkillRel",
                        "predicate": [
                            {
                                "lt": ["skill:rel:rank",2]
                            }
                        ],
                        "value": "rel"
                    },
                    {
                        "label": "PF2E.SkillOcc",
                        "predicate": [
                            {
                                "lt": ["skill:occ:rank",2]
                            }
                        ],
                        "value": "occ"
                    }
```

File: packs/data/feats.db/hellknight-signifer-dedication.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/feats.db/hellknight-signifer-dedication.json]: Good catch, fixed.

Comment (stwlam) [reply packs/data/feats.db/hellknight-signifer-dedication.json]: It doesn't look fixed :(

**Pull request merged**